Source/Menu/Options.cs

```diff
@@ -759,5 +760,94 @@ private void checkPerformanceTuner_Click(object sender, EventArgs e)
             labelPerformanceTunerTarget.Enabled = checkPerformanceTuner.Checked;
         }
 
+
+        #region Help for General Options
+        /// <summary>
+        /// Loads a relevant page from the manual maintained by James Ross's automatic build
+        /// Info icon taken from https://icons8.com/icon/set/info/office--blue
```

Comment:
Icons8's terms of use [dictate](https://intercom.help/icons8-7fb7577e8170/en/articles/4725508-where-do-i-set-the-link) that we need to add a link to them in our About dialog, not just in the source code. Is there an alternative, preferably public domain set we could use?

Reply:
Hi Ryna,
Thanks for spotting this.
Couldn't find any suitable public domain ones, so changed Icon8 ones for homemade.
Best wishes,
Chris